README.md

```diff
@@ -8,7 +8,7 @@ Support is quite minimal and was primarily tailored for our internal use at [Ins
 Packaging is still a work in progress.  
 At the moment the only way to run the app is to to run it from source (either by writing more tests or by creating a main function) but we intend to support the following usages :
 
-- Java CLI (something like `java -jar jeanmichel.jar marathon.json`)
+- Java CLI ( `java -jar jeanmichel.jar "marathon.json appname"`) where appname can be null.
```

Comment:
If we now support Java CLI (thanks to your PR :+1: ), we should probably move it to a separate list as this is the list of what we intend to support in the future.